code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm

```diff
@@ -50,12 +118,17 @@
 /obj/effect/spider/terrorweb/CanPass(atom/movable/mover, turf/target)
 	if(istype(mover, /mob/living/simple_animal/hostile/poison/terror_spider))
 		return 1
+	if(istype(mover, /obj/item/projectile/terrorqueenspit))
+		return 1
 	if(isliving(mover))
 		if(prob(80))
 			to_chat(mover, "<span class='danger'>You get stuck in [src] for a moment.</span>")
 			var/mob/living/M = mover
 			M.Stun(4) // 8 seconds.
 			M.Weaken(4) // 8 seconds.
+			if(iscarbon(mover))
+				spawn(70)
+					qdel(src)
```

Comment:
huh. I guess it works, even though it might lead to janky behavior where someone is caught, another drags them away from the web, and the web spontaneously vanishes.